packages/components/src/Components/ConditionalFilter/Group.js

```diff
@@ -23,13 +23,19 @@ class Group extends Component {
         });
     };
 
-    componentDidUpdate({ selected: prevSelected }) {
-        const { selected } = this.props;
+    componentDidUpdate({ selected: prevSelected, filterBy: prevFilterBy }) {
+        const { selected, filterBy } = this.props;
         if (!isEqual(prevSelected, selected)) {
             this.setState({
                 selected
             });
         }
+
+        if (filterBy !== undefined && prevFilterBy !== filterBy) {
+            this.setState({
```

Comment:
is it possible to only use the filterBy from props?

Reply:
That would require larger update to this component, currently `filterBy` is stored in state and changes whenever user types something in.